packages/apps/fortune/recording-oracle/src/modules/job/job.service.spec.ts

```diff
@@ -136,7 +136,7 @@ describe('JobService', () => {
       (EscrowClient.build as jest.Mock).mockResolvedValue(escrowClient);
 
       const jobSolution: WebhookDto = {
-        escrowAddress: MOCK_ADDRESS,
+        escrowAddress: MOCK_ADDRESS.toLowerCase(),
```

Comment:
If the implementation is correct we should not requiered to pass lowercase in the tests, in should be done internally.